game/platform.cc

```diff
@@ -39,7 +39,7 @@ int Platform::defaultBackEnd() {
 #if (BOOST_OS_WINDOWS)
 extern "C" {
 // For DWORD (see end of file)
-#include "windef.h"
+#include "windows.h"
```

Comment:
Is this a mingw vs msvc thing, or is windef deprecated or what?

Reply:
because a standalone include of windef.h throws an error (in the preprocessor):
#error "No Target Architecture"
This is avoided by simply including the whole headers as intended.

Reply:
a more leightweight alternative would be to include wtypes.h or minwindef.h; but this is probably the most compatible one

Reply:
I think this is resolved?